ports/libxlsxwriter/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "libxlsxwriter",
-  "version-semver": "1.1.4",
+  "version-semver": "1.1.5",
```

```suggestion
  "version": "1.1.5",
```
The time since the last update is too long, `1.1.5` is more suitable to follow the semantic specification of version.
https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#version